tmp/src/scopy_color_editor.cpp

```diff
@@ -110,7 +110,7 @@ ScopyColorEditor::ScopyColorEditor(QApplication *app, QWidget *parent)
 		m_ui->saveBtn->setEnabled(true);
 	});
 	connect(m_ui->restartBtn, &QPushButton::clicked, [=](){
-		adiscope::ApplicationRestarter::triggerRestart();
+		adiscope::ApplicationRestarter::GetInstance()->triggerRestart();
```

Comment:
This is not compiled, it will eventually be removed from the project. No need to update here ..